utkdigitalinitiatives/polk-correspondence-app
# Address PTC-149

**JIRA Ticket**: [PTC-149](https://jirautk.atlassian.net/projects/PTC/issues/PTC-149)

# What does this Pull Request do?

Adds basic JS redirect from the TEI Publisher index page to the actual document that is expected. Necessary for UX on Polk as the app is a single XML.

# What's new?

* Checks URL for current page.
* Redirects to **current href + static Polk home** on explicit match for pathname
* Removed some of the filler on the TEI Publisher index to make redirect more graceful.

# How should this be tested?

1. `ant` and upload.
2. Navigate to http://localhost:8080/exist/apps/polk-papers/ -- expect redirect
3. Navigate to http://localhost:8080/exist/apps/polk-papers -- expect redirect
4. Navigate to http://localhost:8080/exist/apps/polk-papers/polk.xml -- expect NO redirect
5. Do Search -- expect NO redirect
6. Browse table of contents  -- expect NO redirect

# Additional Notes:
Please think of any cases I didn't not include in steps 2-6 above.

# Interested parties
@markpbaggett
